docker-compose.yml

```diff
@@ -25,6 +25,8 @@ services:
       DB_PORT: 5432
       DB_USER: dab
       DB_PASSWORD: dabing
+    stdin_open: true
+    tty: true
```

Comment:
What are these changes for?

Reply:
To be able to attach to a debugging session inside the container, lets say I add a `pdb.set_trace()`